packages/python-google-compute-engine/google_compute_engine/networking/network_daemon.py

```diff
@@ -119,7 +122,9 @@ def _ExtractInterfaceMetadata(self, metadata):
         if self.target_instance_ips:
           ip_addresses.extend(network_interface.get('targetInstanceIps', []))
         interfaces.append(NetworkDaemon.NetworkInterface(
-            interface, ip_addresses, network_interface.get('ip', [])))
+            interface, forwarded_ips=ip_addresses,
+            ip=network_interface.get('ip', None),
+            ipv6='dhcpv6Refresh' in network_interface.keys()))
```

Comment:
this is the metadata key check? should it be case sensitive like this?

Reply:
Yes, since it's defined by internal systems.